dist/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To-do-list</title>
+    <script src="https://kit.fontawesome.com/70657c6fc0.js" crossorigin="anonymous"></script>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+  <script defer src="main.js"></script></head>
+  <body>
+    <section>
+       <div class="title">
+        <h1>Today's To Do</h1>
+        <button type="submit" class="autorenew">
+          <span class="material-symbols-outlined"> autorenew </span>
+        </button>
+      </div>
+
+      <div class="add-list">
+      <form action="">
+        <input placeholder="Add to your list..." id="add-task" />
+      </form>
+    <span class="material-symbols-outlined"> subdirectory_arrow_left </span>
+    </div>
+    
+    <div class="to-do-items"></div>
+    <p class="message"></p>
+    
+    <div class="btn-clear">
+      <button type="submit">Clear all completed</button>
+    </div>
+    
+    </section>
```

Comment:
- [ Optional ] It would be better if you centered the `to-do list` as shown in the video below, currently, the to-do list is aligned to the left leaving a large white space on the right. Please see the screenshot below for more details on this.

## Microverse design
https://www.youtube.com/watch?v=AcUd-_Yjjqg

## Your design
<img src="https://user-images.githubusercontent.com/16392046/214559326-fd69f783-92fa-428d-af0e-dc670274c0dc.png" />